packages/desktop-client/src/components/transactions/MobileTransaction.js

```diff
@@ -99,6 +175,768 @@ function Status({ status }) {
+  //   if (this.lastChildAmount) {
+  //     this.lastChildAmount.focus();
+  //   }
+  // };
+
+  // onDeleteSplit = transaction => {
+  //   let { transactions } = this.state;
+  //   let { data } = deleteTransaction(transactions, transaction.id);
+  //   this.setState({ transactions: data });
+  // };
+
+  // TODO: `renderActions` was used to render swipe actions on split transaction
+  // rows. It enabled you to swipe, revealing a delete action, and if you swiped
+  // all the way it would perform the delete action. The entire swipe was
+  // animated. Swipe actions are a native paradigm and we'll need a web-friendly
+  // replacement.
+  //
+  // renderActions = (progress, dragX) => {
+  //   const trans = dragX.interpolate({
+  //     inputRange: [-101, -100, -50, 0],
+  //     outputRange: [-6, -5, -5, 20],
+  //   });
+  //   return (
+  //     <RectButton
+  //       onPress={this.close}
+  //       style={{
+  //         flex: 1,
+  //         justifyContent: 'center',
+  //         backgroundColor: colors.r4,
+  //       }}
+  //     >
+  //       <Animated.Text
+  //         style={{
+  //           color: 'white',
+  //           textAlign: 'right',
+  //           transform: [{ translateX: trans }],
+  //         }}
+  //       >
+  //         Delete
+  //       </Animated.Text>
+  //     </RectButton>
+  //   );
+  // };
+
+  render() {
+    const {
+      adding,
+      categories,
+      accounts,
+      payees,
+      renderChildEdit,
+      navigation,
+      onDelete,
+    } = this.props;
+    const { editingChild } = this.state;
+    const transactions = this.serializeTransactions(
+      this.state.transactions || [],
+    );
+    const [transaction, ..._childTransactions] = transactions;
+    const { payee: payeeId, category, account: accountId } = transaction;
+
+    // Child transactions should always default to the signage
+    // of the parent transaction
+    let forcedSign = transaction.amount < 0 ? 'negative' : 'positive';
+
+    let account = getAccountsById(accounts)[accountId];
+    let payee = payees && payeeId && getPayeesById(payees)[payeeId];
+    let transferAcct =
+      payee &&
+      payee.transfer_acct &&
+      getAccountsById(accounts)[payee.transfer_acct];
+
+    let descriptionPretty = getDescriptionPretty(
+      transaction,
+      payee,
+      transferAcct,
+    );
+
+    return (
+      // <KeyboardAvoidingView>
+      <View
+        style={{
+          margin: 10,
+          marginTop: 3,
+          backgroundColor: colors.n11,
+          flex: 1,
+          borderRadius: 4,
+
+          // This shadow make the card "pop" off of the screen below
+          // it
+          shadowColor: colors.n3,
+          shadowOffset: { width: 0, height: 0 },
+          shadowRadius: 4,
+          shadowOpacity: 1,
+        }}
+      >
+        <View
+          style={{
+            borderRadius: 4,
+            overflow: 'hidden',
+            flex: 1,
+          }}
+        >
+          <View
```

Comment:
It would be nice to have a back button here to allow quickly exiting the edit screen without saving or having to swipe back.

Reply:
Absolutely! Added via https://github.com/actualbudget/actual/pull/1340/commits/754c574430f1bbbcdc85002c4cbc818d0e13ed5f.